src/CLUBB_core/clubb_api_module.F90

```diff
@@ -1402,6 +1405,10 @@ subroutine advance_clubb_core_api_single_col( gr, &       ! intent(in)
     !$acc end data
     !$acc end data
     !$acc end data
+      
+#ifdef CLUBB_CAM
```

Comment:
Why is this needed?

Reply:
Hing thinks that this was included by accident, but he hasn't tested the code on a GPU, so he's not 100% sure.